### ISSUE TITLE
MID descriptions in other languages

### ISSUE THREAD

**Issue by spandanagella** [openimages/dataset#25]
Hello,

Just wondering if its possible to provide short descriptions of the mid's in other languages? 

For example an entry in dict could like:

"/m/06z6r","swimming"  -> "/m/06z6r","swimming","natación", "Schwimmen"

As freebase api is no longer available and there is strict quota on Google Knowledge graph if you could provide short descriptions in other languages it would be great. I'm currently looking for Spanish and German.

Or if there is a easier way to get the labels in other languages please let me know.

Spandana

**Comment by krasin** [comment]
Hi @spandanagella,

generally, something like that makes sense to implement at some point. Right now, I believe the dataset development has other priorities, like reducing noise in the annotations or adding images with rare entities.

As a somewhat helpful datapoint, I would like to point out that Freebase RDF dumps are still available in various places, like on [Archive.org](https://archive.org/details/freebase-rdf-latest)

**Comment by spandanagella** [comment]
Hi @krasin 

Thank you for getting back. I have figured out a way to do this for now using wiki data mappings with freebase.  However, I see some of the mid's do not have mappings. 

spandana

**Comment by krasin** [comment]
Yes, there could be some entities missing from FreeBase, because its development stopped, but hopefully not too many.

By the way, why do you need this localized mids? I can think of several potential uses, but most of them felt theoretical to me.
